content/guides/pricing-concepts.md

```diff
@@ -0,0 +1,120 @@
+
+Keep in mind that `taxes` outside `price_details` are just grouped summed taxes of the same kind that should be used only for presentation purposes.
+
+The response will contain three top-level attributes that are related to the price:
+
+* `initial_price` - the initial price (without taxes and fees). If you don't apply any discounts, this will be equal to the price for only the rental.
+* `final_price` - the sum of initial price, required fees and excluded taxes.
+* `price_to_pay_now` - if you capture credit card payments in your application, this is the price that you should charge which is calculated based on the payment schedule (rental's balance due date - before that date you should charge the price based on the downpayment, after that the full price (`final_price`)). If you use Live Quote, you don't have to worry how this attribute is calculated.
+
+### 2. Distributed calculation cache - LOS records (recommended for asynchronous flow)
+
+Sometimes you can't perform a live quote before creating a booking, but still need the best approximation of the price. In that case, you can take advantage of [LOS records](http://developers.bookingsync.com/reference/endpoints/los_records/), which are a distributed cache of the prices for the next consecutive 30 days in reference to a given `day` when the booking is created. Most likely this day will be today's date if you use LOS records for price calculation before creating a booking.
+
+The way LOS records' `rates` work is the following: each consecutive rate is the price for the given length of stay (hence Length Of Stay Records). For a one-day stay, the price will be equal to the first rate, for a two-day stay, the price will be equal to the second rate and so forth.
+
+There are three `kind`s of LOS records, but two of them are particularly important:
+
+* `rental_price` - the LOS records will contain the rates for booking's `initial_price` (check Live Quote docs for reference).
+* `final_price ` - the LOS records will contain the rates for booking's `final_price`, which means it covers excluded taxes and required fees as well (check Live Quote docs for reference).
+
+### 3. Rates and rates rules (not recommended).
+
+Besides LOS records, there is one more form of a distributed calculations cache - [rates](http://developers.bookingsync.com/reference/endpoints/rates/). They are in some sense similar to LOS records, but the rates are divided into the periods without a reference to the specific day for which they were calculated. That means that not all [rates rules](http://developers.bookingsync.com/reference/endpoints/rates_rules/) could have been applied when generating them (like `late_booking` or `early_booking`). If you **really** have to use rates, you should apply rates rules that make sense on top of them when calculating the price for a given booking, but even then the price most likely will not be the same as from LOS records or Live Quote.
```

Comment:
> you should apply rates rules that make sense on top of them when calculating the price for a given booking, but even then the price most likely will not be the same as from LOS records or Live Quote.

WRONG: Rates already have pricing rules applied to them

Reply:
I think we should not ask to apply additional rules in this case, way to risky to apply to strong discounts. `late_booking` and `early_booking` are already applied I think, based on cache regeneration day (today) as well.

Reply:
Ok.